docs/sql/statements/create-table.rst

```diff
@@ -161,6 +163,7 @@ to the table as a whole.
 .. SEEALSO::
 
     :ref:`General SQL: Table constraints <table_constraints>`
+    :ref:`CHECK constraint <check_constraint>`
```

Comment:
Should we use a bullet list here and below? Just an idea, if it is not appropriate then just skip my suggestion.

```suggestion
    - :ref:`General SQL: Table constraints <table_constraints>`
    - :ref:`CHECK constraint <check_constraint>`
```

Reply:
yep makes sense :+1: